.travis.yml

```diff
@@ -2,37 +2,57 @@ sudo: true
 language: python
 cache: pip
 python:
-- '3.7'
+  - '3.7'
 env:
   global:
     - AWS_DEFAULT_REGION=eu-west-1
     - AWS_DEFAULT_OUTPUT=json
     - DRIFT_TIER=DEVNORTH
-  matrix:
+    - PATH=$HOME/.local/bin:$PATH
+    - BOTO_CONFIG=/dev/null
     - DRIFT_CONFIG_URL=s3://relib-test/directive-games
+    - DOCKER_USERNAME=nonnib
+    - IMAGE_NAME=directivegames/drift-base
+    - secure: VW2DI3gVIPeZRwsyVHPOIEkRHWlCHr00TAhx0c/Y0OVnyFAQUA/XymNGbDf92/r8T/fQoKjdm9j9zp+XupNiDkMOH0y+uA1poolF/npnqvUIDGIkj67eF5HcyM1QCsie1iCm/JuyMCVFG+XQy6AfhyHyjvvGFMLxInADvgmsQMLYuLU/5ZdsLcv7fJcPGMTjxqtcEf017ZbSAShDYmag91jNdKOTU50C16c+m2dEskDM5JDhWvxeJ6mkdhmdtHRAIGkG4Ix+nd0dzBFhCd8V8TjUW/kp9XmAbjPkHbu6h3UKAph8bkW6O+wmHkEnmaDRsSuKTv6QFwUEZgzodacJ1azYjSXoigpKlTTGp4AxDxMD23qix8Wr1KxGIGKXRQSkLWaWym2G6Gri7NEHuJidgZQ/RQJA6k6aur5xB/bIdnGxtqmiv8sJlWuvw8HnyEMv1i8BViFgGh7CO1fpfrPokbi6wMjENc6ObmzoH6u+MeFyZyolco6eJ2JeivgvWUGnfitZPPuTsxg+gaUj4ei8D9ilUQZSKLf4dmrXkqrii5eRg+hdS68lno2kcZIPnMbh23m6KjrHofFanAfm/DOM0bcN0FtqLlrw4B2JOUYuB5lHnTCqJtAxEv3Rpx8DHG0VNzZEdW2Rjp5VdRa0Ea5hy63L7mPxHlIRKzRsMLPz+N4=
```

Comment:
What's this? Secure what? This is a public repo. Please remove, *and* change it.

Reply:
'secure' is a travis paradigm for encrypting information and should be safe in a public repo. Some information here. https://docs.travis-ci.com/user/encryption-keys/